### DISCUSSION TITLE
why we map lt function  to lessthan function？

### DISCUSSION THREAD

**Discussion by HaoChen-ch** [apache/gluten#6247] [Q&A]
https://github.com/apache/incubator-gluten/blob/3a42e8fbd3797390a554839ef10e6f9b073460d6/cpp/velox/substrait/SubstraitParser.cc#L382
It seems that lt function is different with lessthan function .and lt use simd instruct to accelerate fliter ,why not use le ? same as lte,gt, gt2, ...

**Comment by philo-he** [comment]
@HaoChen-ch, maybe, the mapping is for fixing semantic issues. You can try to remove them to see whether Spark UTs (already imported into Gluten) can pass.
